include/crypto/dh.h

```diff
@@ -24,8 +25,10 @@ void dh_cache_named_group(DH *dh);
 
 FFC_PARAMS *dh_get0_params(DH *dh);
 int dh_get0_nid(const DH *dh);
-int dh_ffc_params_fromdata(DH *dh, const OSSL_PARAM params[]);
+int dh_params_fromdata(DH *dh, const OSSL_PARAM params[]);
```

Comment:
Should these things have `ossl_` prefixes?

Reply:
*shrug*
"DH" and "dh" is a pre-existing API prefix, I personally find it quite confusing if we end up with some `dh_blah` and some `ossl_dh_bleh` depending on how long they have been around.
Buf more surprisingly, I thought that there had already been a rename fest PR for DH functions...
Either way, I consider the rename stuff off topic for *this* PR.  It would be silly to just rename one function if we should really rename a whole set.